src/service/calculator/calculator_report.py

```diff
@@ -104,9 +113,31 @@ def get_rule_of_40(self, company: dict, company_revenue: int) -> dict:
             "revenue": self.calculator.calculate_base_metric(company_revenue),
         }
 
-    def anonymized_company(self, company: dict, allowed_companies: list) -> None:
+    def anonymized_companies_metrics(
+        self, access: bool, companies: dict, allowed_companies: list
+    ) -> None:
+        gross_profit_records = []
+
+        for id in companies:
+            company = companies[id]
+            gross_profit_records.append(company.get("gross_profit"))
```

Comment:
Use list comprehension instead
`gross_profit_records = [companies[company_id].get("gross_profit") for company_id in companies]`

Reply:
I changed it